network/src/protocols/rpc/mod.rs

```diff
@@ -466,23 +471,26 @@ async fn handle_outbound_rpc_inner(
         raw_request: Vec::from(req_data.as_ref()),
     });
 
-    // Start timer to collect RPC latency.
-    let _timer = counters::LIBRA_NETWORK_RPC_LATENCY.start_timer();
-
     // Send outbound request to peer_handle.
     trace!(
         "Sending outbound rpc request with request_id {} to peer: {:?}",
         request_id,
         peer_id.short_str()
     );
+    let prototol_id_descriptor = protocol.to_string();
+    // Start timer to collect RPC latency.
+    let timer = counters::LIBRA_NETWORK_RPC_LATENCY
+        .with_label_values(&[REQUEST_LABEL, &prototol_id_descriptor, &peer_id_str])
```

Comment:
nit: use `protocol.as_str()` here instead of `protocol.to_string()`

Reply:
nit: replace the other `peer_id.short_str()` with `peer_id_str` in this fn

Reply:
done!